src/mjolnir/shortcutbuilder.cc

```diff
@@ -400,8 +398,15 @@ uint32_t AddShortcutEdges(GraphReader& reader,
       DirectedEdge newedge = *directededge;
       uint32_t length = newedge.length();
 
-      // For computing weighted density along the shortcut
+      // For computing weighted density and total turn duration along the shortcut
       float average_density = length * newedge.density();
+      uint32_t const speed = tile->GetSpeed(directededge, kNoFlowMask);
```

Comment:
do we really need to call `GetSpeed`? in this case it equals to `directededge->speed()`

Reply:
we dont but its convenient to do so because it is what actual sif::DynamicCost and the others do. if for some reason we changed how GetSpeed worked underneath, then we'd get that change here as well as in the costing. the point is that we do the same thing the costing does so that this number and the number costing use are in sync. otherwise i agree, its very heavy for something so trivial. its more about making sure stuff stays in sync.